### ISSUE TITLE
"argument" should be "parameter" here?

### ISSUE THREAD

**Issue by JoeHetfield** [getify/Functional-Light-JS#95]
Ch2, line 277:

> Do you spot the `[ .. ]` brackets around the argument list now? That's array destructuring.

And the code snippet in question:

```js
function foo( [x,y,...args] = [] ) {
	// ..
}

foo( [1,2,3] );
```

I think the destructuring would happen at ` [x,y,...args] ` part, so the "argument list" in the text should be "parameter list".

**Comment by getify** [comment]
Yep, good catch. PR?

**Comment by JoeHetfield** [comment]
PR submitted.

**Comment by getify** [comment]
fixed in #96
